### PR TITLE
Release updates for version 1.0.7

### PR DESCRIPTION
**Pull request by kanika1894**
The changes related to v1.0.6 were done directly in the release branch as the entire master batch wasn't to be released then.
Now updating the current master with v1.0.6 changes we well.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,15 @@ All notable changes to the Deploy to Azure extension will be documented in this
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/). Versioning follows an internal Azure DevOps format that is not compatible with SemVer.
 
+## 1.0.7
+### Added
+- Integrated with repo analysis and template service
```
Comment (bishal-pdMSFT) [inline CHANGELOG.md:8]: should we be more verbose here? YOu can say that not the extension does better analysis of the repo and based on that analysis it suggests a weighted list of more accurate templates. It uses backend services for both repo analysis and template management

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,15 @@ All notable changes to the Deploy to Azure extension will be documented in this
...
+- If not a remote repo, have a provision of creating github repo
+
+## 1.0.6
```
Comment (bishal-pdMSFT) [inline CHANGELOG.md:11]: Did we release this version? If no, then fold it into 1.0.7

Comment (kanika1894) [reply CHANGELOG.md:11]: we released this version, changes were done directly in release branch

File: CHANGELOG.md (hunk 1 above)
Comment (kanika1894) [reply CHANGELOG.md]: Have updated the text by cross-checking with Pulkit

File: src/configure/configure.ts [hunk 3]
```diff
@@ -162,13 +162,14 @@ class Orchestrator {
...
             await this.getSourceRepositoryDetails();
-            await this.getAzureSession();
+            this.inputs.azureSession = getAzureSession();
             let repoAnalysisResult = await this.getRepositoryAnalysis();
             await this.getSelectedPipeline(repoAnalysisResult);
 
             try {
                 if (!resourceNode) {
                     this.context['rightClickScenario'] = false;
+                    await this.getAzureSubscription();
```
Comment (bishal-pdMSFT) [inline src/configure/configure.ts:172]: after subscription is selected we should update the `this.azureSession` to ensure that we have right token

File: src/configure/configure.ts [hunk 4]
```diff
@@ -465,7 +466,6 @@ class Orchestrator {
         });
         let selectedSubscription: QuickPickItemWithData = await this.controlProvider.showQuickPick(constants.SelectSubscription, subscriptionList, { placeHolder: Messages.selectSubscription }, TelemetryKeys.SubscriptionListCount);
         this.inputs.subscriptionId = selectedSubscription.data.subscription.subscriptionId;
-        this.inputs.azureSession = getSubscriptionSession(this.inputs.subscriptionId);
```
Comment (bishal-pdMSFT) [inline src/configure/configure.ts:468]: this line is required

File: src/configure/configure.ts [hunk 5]
```diff
@@ -162,13 +162,14 @@ class Orchestrator {
 
         if (this.continueOrchestration) {
             await this.getSourceRepositoryDetails();
-            await this.getAzureSession();
+            this.inputs.azureSession = getAzureSession();
```
Comment (bishal-pdMSFT) [inline src/configure/configure.ts:165]: for web app right click scenario, we should pass the subscriptionId as it is already known and we will not be explicitly setting subscription later.

File: src/configure/configure.ts (hunk 4 above)
Comment (kanika1894) [reply src/configure/configure.ts:468]: added

File: src/configure/configure.ts (hunk 3 above)
Comment (kanika1894) [reply src/configure/configure.ts:172]: added condition

File: src/configure/configure.ts (hunk 5 above)
Comment (kanika1894) [reply src/configure/configure.ts:165]: added condition

**Pull request merged**